### ISSUE TITLE
Can the integration be used with the Worcester Wave

### ISSUE THREAD

**Issue by toeknee-uk** [ksya/ha-nefiteasy#254]
Hi,
I was wondering if the integration would or should work with Worcester Wave thermostat.  I understand that it is a rebadged nefit and as a subsidiary of Bosch may I would hope use the same backend services.  I have tried to login but the integration shows the combination to be wrong.  Would be great to have it in home assistant as I find the native app to be terrible.  Thanks

**Comment by m11tch** [comment]
Looks like a rebrand of the 'nefit easycontrol' rather then the 'nefit easy' I believe that also uses a different app and thus a different home-assistant integration.

As I only have a 'nefit easy' myself, I'm not quite sure which integration you'd need. But searching for 'nefit easycontrol' should probably get you in the right direction :)

**Comment by toeknee-uk** [comment]
Thanks for coming back to me so quickly - I have downloaded an alternative integration but am still getting an issue I'll set up debugging and fingers crossed can get it working!  Cheers

**Comment by RobBie1221** [comment]
You have this one?
https://www.worcester-bosch.co.uk/professional/products/boiler-controls/wave-smart-control

If so, that looks pretty similar to the Nefit Easy and the integration might work. The main question is whether you are using the same server as the Nefit Easy. The server address is hardcoded in the supporting library.

**Comment by toeknee-uk** [comment]
Yes that's the one.  It truly is a horrible app setting up the timer is impenetrable. I had a Tado before which was amazing but when fitting the boiler they extend the warranty by three years for using their thermostat.  I wonder if it's a different server. I'm new to Home Assistant and adding debugging looks a little daunting but will maybe set up a second test setup of HA to play with.

**Comment by RobBie1221** [comment]
Did you remove the spaces from the "Access Key" that you entered? Just to check if you are not making a basic mistake?

**Comment by toeknee-uk** [comment]
I did remove the spacing and set a new 8 digit password with no special characters.  On the other integration it shows a "unknown error"

**Comment by bsimmo** [comment]
> Hi, I was wondering if the integration would or should work with Worcester Wave thermostat. I understand that it is a rebadged nefit and as a subsidiary of Bosch may I would hope use the same backend services. I have tried to login but the integration shows the combination to be wrong. Would be great to have it in home assistant as I find the native app to be terrible. Thanks

Yes, I've been using it for a few years now
